extensions/blocks/rating-star/index.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import { __, _x } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+import save from './save';
+import { StarIcon, StarBlockIcon } from './icon';
+import './editor.scss';
+import './style.scss';
+
+export const name = 'jetpack/rating-star';
```

Comment:
You'll need to name it `rating-star` or it won't be made available.

```suggestion
export const name = 'rating-star';
```

Reply:
Fixed at https://github.com/Automattic/jetpack/pull/14091/commits/35310a701e1e98c72febdccfa7f9b6130df90f8f